avl_tree/AVL_Tree.java

```diff
@@ -0,0 +1,294 @@
+import java.util.Scanner;
+
+public class AVL_Tree {
+
+	public static void main(String[] args) {
+		AVL avl = new AVL();
+		int a;
+		Scanner inp = new Scanner(System.in);
+		while (true) {

```

Comment:
@Hsankesara  I'm saying not to use a bool variable did I?  I'm saying to hardcode inputs please see other implementation of DS you will get the idea of what I'm trying to say here